breadbox-client/breadbox_facade/client.py

```diff
@@ -368,134 +368,12 @@ def get_dimension_type(self, name: str):
         breadbox_response = get_dimension_type_client.sync_detailed(name=name, client=self.client)
         return self._parse_client_response(breadbox_response)
 
-    def add_feature_type(
-        self,
-        name: str,
-        id_column: str,
-        metadata_df: Optional[pd.DataFrame] = None,
-        taiga_id: Optional[str] = None,
-        annotation_type_mapping: Optional[dict] = None,
-        id_mapping: Optional[dict] = None,
-        properties_to_index: Optional[list[str]] = None,
-    ) -> FeatureTypeOut:
-        metadata_file = File(
-            payload=metadata_df.to_csv(index=False),
-            file_name=f"{name}.csv",
-            mime_type="text/csv",
-        )
-        type_mapping = AnnotationTypeMap.from_dict(annotation_type_mapping) if annotation_type_mapping else None
-        params = BodyAddFeatureType(
-            name=name,
-            id_column=id_column,
-            metadata_file=metadata_file,
-            taiga_id=taiga_id,
-            annotation_type_mapping=type_mapping,
-            id_mapping=IdMapping.from_dict(id_mapping) if id_mapping else UNSET,
-            properties_to_index=properties_to_index if properties_to_index else UNSET,
-        )
-        breadbox_response = add_feature_type_client.sync_detailed(client=self.client, multipart_data=params)
-        return self._parse_client_response(breadbox_response)
-
-    def add_sample_type(
```

Comment:
Heads up, these old functions are still being used by the old `sync_metadata_to_breadbox` utility, so these deletions may break the `portal-backend` type checker (once the portal starts referencing this new version of the breadbox client). 

It seems like that `sync_metadata_to_breadbox` process has been broken for a little while now (I was trying to use it this afternoon and was disappointed),  and I'm surprised no one else has complained about it, so I added a topic to our breadbox discussion list https://app.asana.com/0/1206925945720127/1208261191969013/f